.typos.toml

```diff
@@ -4,6 +4,7 @@ extend-exclude = [
     "google/cloud/bigtable/internal/readrowsparser_acceptance_tests.inc",
     # The source proto files have one or more typos in their comments
     "google/cloud/accessapproval/access_approval_client.h",
+    "google/cloud/channel/cloud_channel_client.h",
```

Comment:
Consider sending CLs to fix the typos upstream, I just merged my fixes for storage_transfer_client.h

Reply:
It's not clear that it's a real typo. It complained about "add-ons" (the ons part). Maybe it's a `typos` bug. Not sure. In a way, it may be good to teach typos to ignore all auto-generated libraries.

Reply:
The comments go straight to our Doxygen documentation.  The number of typos is small, but I would rather not ignore them and fix the problems upstream.  Less embarrassing for everyone involved.  It would be nice if the protos in googleapis/googleapis used some kind of typo detector too.

Reply:
Makes sense. I'll keep that in mind. In this case, `typos` complained about "Add-ons", which I think is spelled correctly; it's likely a bug in `typos`.

Reply:
Can't we add an `extend-words` entry to our `.typos.toml` file?

Reply:
Generally, we could do that. In this case, however, the typo is actually "ons" from the word "add-ons". So if we used `extend-words` we'd have to teach it that "ons" is a valid word, which is generally not true. Extending the allowed words to allow "add-ons" does not work.

FTR, the actual output was:
```
  --> ./google/cloud/channel/cloud_channel_client.h:813:10
    |
813 |   /// add-ons, or entitlements for Google Cloud's development platform.
    |           ^^^
    |
... 0.784 seconds
```